custom/icds_reports/management/commands/populate_local_db_with_icds_dashboard_test_data.py

```diff
@@ -30,5 +33,33 @@ def handle(self, *args, **options):
                     '=================================================\n'
                 )
                 raise
+            _rename_locations()
         else:
             print('operation canceled')
+
+
+def _rename_locations():
+    # we need to use real location names because the map depends on names as keys
+    new_name_map = (
+        ('state_name', 'st1', 'Bihar'),
```

Comment:
I think it can also be  achieved by setting map_location_name in metadata here https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/tests/agg_tests/agg_setup.py#L63

Reply:
I don't think so, because the `awc_locations` table is simply populated from the data fixture, no? Let me know if I'm missing something, but I originally tried changing it there but it wasn't making it's way into the aggregation tables.